pkg/balancer-js/src/utils/errors.ts

```diff
@@ -66,7 +66,7 @@ const balancerErrorCodes: Record<string, string> = {
   '337': 'UNAUTHORIZED_EXIT',
   '338': 'MAX_MANAGEMENT_SWAP_FEE_PERCENTAGE',
   '339': 'UNHANDLED_BY_MANAGED_POOL',
-  '340': 'UNHANDLED_BY_PHANTOM_POOL',
+  '340': 'UNHANDLED_BY_STABLE_POOL',
```

Comment:
Let's keep the old name - this error existed in previous versions of the phantom pool.